src/array_buffer.rs

```diff
@@ -242,6 +243,18 @@ pub unsafe extern "C" fn backing_store_deleter_callback(
   drop(b);
 }
 
+pub unsafe extern "C" fn vec_deleter_callback(
+  data: *mut c_void,
+  byte_length: usize,
+  _deleter_data: *mut c_void,
+) {
+  drop(Vec::from_raw_parts(
+    data as *mut u8,
+    byte_length,
+    byte_length,
```

Comment:
One of the safety requirements of `Vec::from_raw_parts` is that the passed capacity must be the vector's capacity. The capacity should probably be leaked as a `Box<usize>` and passed as the deleter data.